Fix errors

Linter is pointing out the following errors:
```
ERROR in ./app/router.js

/Users/raulgarcia/FastMonkeys/kore/app/router.js
  3:7   error  There should be no space after '{'   space-in-brackets
  3:63  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/routes.js

/Users/raulgarcia/FastMonkeys/kore/app/routes.js
  4:7   error  There should be no space after '{'   space-in-brackets
  4:25  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/App.js

/Users/raulgarcia/FastMonkeys/kore/app/App.js
  5:7   error  There should be no space after '{'   space-in-brackets
  5:22  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/components/Navbar.js

/Users/raulgarcia/FastMonkeys/kore/app/components/Navbar.js
  4:7   error  There should be no space after '{'   space-in-brackets
  4:14  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/stores/SearchStore.js

/Users/raulgarcia/FastMonkeys/kore/app/stores/SearchStore.js
  5:7   error  There should be no space after '{'   space-in-brackets
  5:22  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/components/SearchResultsTable.js

/Users/raulgarcia/FastMonkeys/kore/app/components/SearchResultsTable.js
  4:7   error  There should be no space after '{'   space-in-brackets
  4:14  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/stores/SchoolStore.js

/Users/raulgarcia/FastMonkeys/kore/app/stores/SchoolStore.js
  6:7   error  There should be no space after '{'   space-in-brackets
  6:22  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/stores/UIStore.js

/Users/raulgarcia/FastMonkeys/kore/app/stores/UIStore.js
  5:7   error  There should be no space after '{'   space-in-brackets
  5:22  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/core/AppDispatcher.js

/Users/raulgarcia/FastMonkeys/kore/app/core/AppDispatcher.js
  3:7   error  There should be no space after '{'   space-in-brackets
  3:20  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)


ERROR in ./app/components/NavTab.js

/Users/raulgarcia/FastMonkeys/kore/app/components/NavTab.js
  4:7   error  There should be no space after '{'   space-in-brackets
  4:14  error  There should be no space before '}'  space-in-brackets

✖ 2 problems (2 errors, 0 warnings)
```

--- a/app/App.js
+++ b/app/App.js
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import DocumentTitle from 'react-document-title';
-import { RouteHandler } from 'react-router';
+import {RouteHandler} from 'react-router';
 import './styles/app.less';
 import Navbar from './components/Navbar';
 

--- a/app/components/NavTab.js
+++ b/app/components/NavTab.js
@@ -1,7 +1,7 @@
 'use strict';
 
 import React from 'react';
-import { Link } from 'react-router';
+import {Link} from 'react-router';
 
 class NavTab extends React.Component {
   render() {

--- a/app/components/Navbar.js
+++ b/app/components/Navbar.js
@@ -1,7 +1,7 @@
 'use strict';
 
 import React from 'react';
-import { Link } from 'react-router';
+import {Link} from 'react-router';
 import NavTab from './NavTab';
 
 class Navbar extends React.Component {

--- a/app/components/SearchResultsTable.js
+++ b/app/components/SearchResultsTable.js
@@ -1,7 +1,7 @@
 'use strict';
 
 import React from 'react';
-import { Link } from 'react-router';
+import {Link} from 'react-router';
 import Loader from 'react-loader';
 
 class SearchResultsTable extends React.Component {

--- a/app/components/__tests__/InfoRow-test.js
+++ b/app/components/__tests__/InfoRow-test.js
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import TestUtils from 'react/lib/ReactTestUtils';
-import { expect } from 'chai';
+import {expect} from 'chai';
 import InfoRow from '../InfoRow';
 
 describe('InfoRow', function() {

--- a/app/components/__tests__/SchoolTitle-test.js
+++ b/app/components/__tests__/SchoolTitle-test.js
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import TestUtils from 'react/lib/ReactTestUtils';
-import { expect } from 'chai';
+import {expect} from 'chai';
 import SchoolTitle from '../SchoolTitle';
 
 describe('SchoolTitle', function() {

--- a/app/core/AppDispatcher.js
+++ b/app/core/AppDispatcher.js
@@ -1,6 +1,6 @@
 'use strict';
 
-import { Dispatcher } from 'flux';
+import {Dispatcher} from 'flux';
 import PayloadSources from '../constants/PayloadSources';
 
 const isProduction = process.env.NODE_ENV === 'production';

--- a/app/pages/NotFoundPage.js
+++ b/app/pages/NotFoundPage.js
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import DocumentTitle from 'react-document-title';
-import { Link } from 'react-router';
+import {Link} from 'react-router';
 
 class NotFoundPage extends React.Component {
 

--- a/app/router.js
+++ b/app/router.js
@@ -1,6 +1,6 @@
 'use strict';
 
-import { create as createRouter, HistoryLocation, HashLocation } from 'react-router';
+import {create as createRouter, HistoryLocation, HashLocation} from 'react-router';
 import routes from './routes';
 
 export default createRouter({

--- a/app/routes.js
+++ b/app/routes.js
@@ -1,7 +1,7 @@
 'use strict';
 
 import React from 'react';
-import { Route, Redirect, NotFoundRoute } from 'react-router';
+import {Route, Redirect, NotFoundRoute} from 'react-router';
 import App from './App';
 import SearchPage from './pages/SearchPage';
 import AboutPage from './pages/AboutPage';

--- a/app/stores/SchoolStore.js
+++ b/app/stores/SchoolStore.js
@@ -3,7 +3,7 @@
 import _ from 'lodash';
 import AppDispatcher from '../core/AppDispatcher';
 import ActionTypes from '../constants/ActionTypes';
-import { EventEmitter } from 'events';
+import {EventEmitter} from 'events';
 
 const CHANGE_EVENT = 'change';
 let _schools = {};

--- a/app/stores/SearchStore.js
+++ b/app/stores/SearchStore.js
@@ -2,7 +2,7 @@
 
 import AppDispatcher from '../core/AppDispatcher';
 import ActionTypes from '../constants/ActionTypes';
-import { EventEmitter } from 'events';
+import {EventEmitter} from 'events';
 
 const CHANGE_EVENT = 'change';
 let _fetchingData = false;

--- a/app/stores/UIStore.js
+++ b/app/stores/UIStore.js
@@ -2,7 +2,7 @@
 
 import AppDispatcher from '../core/AppDispatcher';
 import ActionTypes from '../constants/ActionTypes';
-import { EventEmitter } from 'events';
+import {EventEmitter} from 'events';
 
 const CHANGE_EVENT = 'change';
 let _schoolTimelineYear = null;

